examples/container-registries/gcp/ts/index.ts

```diff
@@ -12,11 +12,12 @@ const registryInfo = undefined; // use gcloud for authentication.
 
 // Build and publish the image.
 const image = new docker.Image("my-image", {
-    build: "./app",
+    build: {
+        context: "app"
+    },
     imageName: imageName,
     registry: registryInfo,
```

Comment:
comment on old code: we dont need this bc it will infer the registry info from the image name